pkg/solver/server.go

```diff
@@ -520,50 +524,113 @@ func (solverServer *solverServer) updateTransactionsMediator(payload data.DealTr
 *
 */
 
-func (solverServer *solverServer) downloadFiles(res corehttp.ResponseWriter, req *corehttp.Request) {
+type EmptyResponse struct{}
+
+func (solverServer *solverServer) downloadFiles(res corehttp.ResponseWriter, req *corehttp.Request) (EmptyResponse, error) {
```

Comment:
If we removed the EmptyResponse, then we'd just need to change the method signature to look like this (essentially becoming a void func)

```suggestion
func (solverServer *solverServer) downloadFiles(res corehttp.ResponseWriter, req *corehttp.Request) error {
```

Reply:
We decided to stay with `EmptyResponse` because our `http.GetHandler` wrapper requires a type to fill the generic.